examples/04-Icepak/Sherlock_Example.py

```diff
@@ -181,6 +194,30 @@
 
 ipk.assign_priority_on_intersections()
 
+###############################################################################
+# Compute power Budget
+# ~~~~~~~~~~~~~~~~~~~~~~~
+
+power_budget, total = ipk.post.power_budget("W" )
+print(total)
+
+###############################################################################
+# Compute power Budget
+# ~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
all title should have a number of simbols under it equal to the size of the line
```suggestion
# Compute power Budget
# ~~~~~~~~~~~~~~~~~~~~
```

Reply:
Also, the first letter of the first word in a title is capitalized, but all other letters in the title that are not proper nouns are lower case. Here, "Budget" should be "budget".